library/std/src/sys/mod.rs

```diff
@@ -34,6 +34,9 @@ cfg_if::cfg_if! {
     } else if #[cfg(target_arch = "bpf")] {
         mod bpf;
         pub use self::bpf::*;
+    } else if #[cfg(target_arch = "sbf")] {
+        mod sbf;
+        pub use self::sbf::*;
```

Comment:
Same here, I think we should probably rename sys/bpf to sys/sbf, then `use self::sbf::*` if target_arch = "bpf". I'm worried that having this stuff duplicated is going to be a bit of a burden.